### DISCUSSION TITLE
Changing 1st/3rd person

### DISCUSSION THREAD

**Discussion by snej** [bendichter/tenseflow#14] [Ideas]
Over [here](https://github.com/bendichter/tenseflow/issues/5#issuecomment-750542283)) I had asked:

> Would it be feasible to make a similar tool to convert between first and third person? I.e. change "I pick up my hammer" to "He picks up his hammer." The obvious parameter is which sex pronouns (he/she/they); and going from 3rd-->1st one ought to be able to give name(s) for the POV person, so given "Elena picks up her hammer" you'd need to tell it to treat "Elena" as though it were "her".

I implemented 1st⟶3rd person over Xmas break. It works pretty well, with a few glitches where the parser gets a verb's tense wrong (esp. where the 1st person and the infinitive are the same, e.g. "have") or where the `conjugate` function  changes a verb's tense when switching person. But it's good enough that I've used it on about 4,000 words of a story I'm writing, and I only had to fix a handful of mistakes per thousand words.

I'm not sure if you would want to add this to the existing tool, since it's a different function. If so, I could prepare a PR.

**Comment by bendichter** [comment]
cool! Yeah, let's add it!
